.github/workflows/ci-build-macos.yaml

```diff
@@ -46,17 +46,19 @@ jobs:
         uses: actions/cache@v2
         with:
           path: |
-            ~/.m2/repository
+            ~/.m2/repository/*/*/*
             !~/.m2/repository/org/apache/pulsar
-            !~/.m2/.gradle-enterprise
-          key: ${{ runner.os }}-maven-dependencies-${{ hashFiles('**/pom.xml') }}
+          key: ${{ runner.os }}-m2-dependencies-all-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-m2-dependencies-all-
 
       - name: Set up JDK 1.8
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
 
       - name: Replace maven's wagon-http version
+        if: ${{ steps.changes.outputs.all_count > steps.changes.outputs.docs_count }}
```

Comment:
Can you help me undestand how this `if` statement work?

Reply:
good catch, that's a copy-paste error. I'll revisit it.

Reply:
resolved